From a380fb04a9c26a348cc43e688e6a70e3db8bff46 Mon Sep 17 00:00:00 2001
From: nxa13443 <chaofan.huang@nxp.com>
Date: Wed, 28 Mar 2018 20:40:49 +0800
Subject: [PATCH 3546/5242] MLK-17933[IMX8QXP B0] Change compatible name for
 B0 encoder and decoder

commit  8d8f6312260703da2d33b652ac980a520d45b4ed from
https://source.codeaurora.org/external/imx/linux-imx.git

Change compatible name for B0 encoder and decoder in fsl-imx8qxp.dtsi

Signed-off-by: nxa13443 <chaofan.huang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi |    4 ++--
 drivers/mxc/vpu-decoder-b0/vpu_b0.c            |    4 ++--
 drivers/mxc/vpu-encoder-b0/vpu_encoder_b0.c    |    4 ++--
 3 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi
index dbaa7d4..5cd9483 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi
@@ -2690,7 +2690,7 @@
 	};
 
 	vpu_decoder: vpu_decoder@2c000000 {
-		compatible = "nxp,imx8qm-vpu-decoder", "nxp,imx8qxp-vpu-decoder";
+		compatible = "nxp,imx8qm-b0-vpudec", "nxp,imx8qxp-b0-vpudec";
 		boot-region = <&decoder_boot>;
 		rpc-region = <&decoder_rpc>;
 		reg = <0x0 0x2c000000 0x0 0x1000000>;
@@ -2704,7 +2704,7 @@
 	};
 
 	vpu_encoder: vpu_encoder@2d000000 {
-		compatible = "nxp,imx8qm-vpu-encoder", "nxp,imx8qxp-vpu-encoder";
+		compatible = "nxp,imx8qm-b0-vpuenc", "nxp,imx8qxp-b0-vpuenc";
 		boot-region = <&encoder_boot>;
 		rpc-region = <&encoder_rpc>;
 		reg = <0x0 0x2d000000 0x0 0x1000000>;
diff --git a/drivers/mxc/vpu-decoder-b0/vpu_b0.c b/drivers/mxc/vpu-decoder-b0/vpu_b0.c
index 7a4274d..ce33722 100755
--- a/drivers/mxc/vpu-decoder-b0/vpu_b0.c
+++ b/drivers/mxc/vpu-decoder-b0/vpu_b0.c
@@ -2641,8 +2641,8 @@ static int vpu_resume(struct device *dev)
 };
 
 static const struct of_device_id vpu_of_match[] = {
-	{ .compatible = "nxp,imx8qm-vpu-decoder", },
-	{ .compatible = "nxp,imx8qxp-vpu-decoder", },
+	{ .compatible = "nxp,imx8qm-b0-vpudec", },
+	{ .compatible = "nxp,imx8qm-b0-vpudec", },
 	{}
 }
 MODULE_DEVICE_TABLE(of, vpu_of_match);
diff --git a/drivers/mxc/vpu-encoder-b0/vpu_encoder_b0.c b/drivers/mxc/vpu-encoder-b0/vpu_encoder_b0.c
index fd8e3f2..fe3886d 100644
--- a/drivers/mxc/vpu-encoder-b0/vpu_encoder_b0.c
+++ b/drivers/mxc/vpu-encoder-b0/vpu_encoder_b0.c
@@ -2050,8 +2050,8 @@ static int vpu_resume(struct device *dev)
 };
 
 static const struct of_device_id vpu_of_match[] = {
-	{ .compatible = "nxp,imx8qm-vpu-encoder", },
-	{ .compatible = "nxp,imx8qxp-vpu-encoder", },
+	{ .compatible = "nxp,imx8qm-b0-vpuenc", },
+	{ .compatible = "nxp,imx8qm-b0-vpuenc", },
 	{}
 }
 MODULE_DEVICE_TABLE(of, vpu_of_match);
-- 
1.7.9.5

